Source/FullScreenPokemon.ts

```diff
@@ -1399,7 +1410,7 @@ module FullScreenPokemon {
                 thing.FSP.stopCycling(thing);
                 return true;
             } else {
-                return thing.FSP.startCycling(thing);
+                return thing.FSP.startCycling(thing, true);
```

Comment:
Can you split this and line 1400/1411 to not return the result of start/stopCycling calls? `stopCycling` is supposed to return void.